src/sentry/rules/processing/delayed_processing.py

```diff
@@ -120,7 +120,9 @@ def _generate_unique_queries(
         # We will later compare the first query results against the second query to calculate
         # a percentage for percentage comparison conditions.
         comparison_interval = condition_data.get("comparisonInterval", DEFAULT_COMPARISON_INTERVAL)
-        second_comparison_query = unique_queries[0]._replace(comparisonInterval=comparison_interval)
+        first_query_copy = unique_queries[0]._asdict()
+        first_query_copy["comparison_interval"] = comparison_interval
```

Comment:
**nit**: on the names that `first_query_copy` is the same thing as `second_query` and it might be a little easier to grok that we're building the second query rather than naming it a copy.

```python
second_query_data = deepcopy(unique_queries[0])
second_query_data["comparison_interval"] = comparison_interval
unique_queries.append(UniqueConditionQuery(**second_query_data))
```
This would reduce 4 lines to 3, and reads a little easier. we also don't reference the `second_query_comparison` later, so we can just append it to the array.